src/main/java/uk/gov/hmcts/reform/em/annotation/rest/ErrorControllerImpl.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.em.annotation.rest;
+
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
```

Comment:
As discussed please add in the below,

1. Java doc to mention why adding a controller will help capture the exception.
2. Unit test for this controller so that Sonar will pass on master build. As any new class needs to have 80% code coverage.
3. Rename the controller to EmErrorController.
4. Log the error to ensure this method is invoked.